.github/workflows/general-ci.yml

```diff
@@ -18,3 +18,25 @@ jobs:
 
       - name: Verify formatting
         run: npm run format-check
+
+  dist-up-to-date:
```

Comment:
This checks that `dist/*` up-to-date. This is what's actually "shipped" as entrypoint in the action.

Reply:
The build failed. Do you know if `ncc` builds are reproducible?

Reply:
Do you mean idempotent? As in: same source same output?

Reply:
Yes

Reply:
It's failing for different reasons:

```
lerna ERR! npm run package stderr:
sh: 1: ncc: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! @exercism/issue-receiver@1.0.0-DEV package: `ncc build`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the @exercism/issue-receiver@1.0.0-DEV package script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-12-29T01_13_01_664Z-debug.log

lerna ERR! npm run package exited 1 in '@exercism/issue-receiver'
lerna WARN complete Waiting for 2 child processes to exit. CTRL-C to exit immediately.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! root@ package: `lerna run package`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the root@ package script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-12-29T01_13_01_697Z-debug.log
```

`ncc` is a dev dependency in each package and locally it works, do you know what causes this?